src/ESM.sol

```diff
@@ -37,22 +38,30 @@ interface DenyLike {
 }
 
 contract ESM {
+    mapping (address => uint256) public wards; // auth
```

Comment:
I'm being probably too pedantic today lol, but I'd group the storage variables together whatever is moving this one right after the immutables or moving the immutables after the last storage variable.